include/receiver_cadmium.hpp

```diff
@@ -0,0 +1,116 @@
+/**
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#ifndef _BOOST_SIMULATION_PDEVS_RECEIVER_HPP_
+#define _BOOST_SIMULATION_PDEVS_RECEIVER_HPP_
+
+
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <limits>
+#include <math.h> 
+#include <assert.h>
+#include <memory>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <chrono>
+#include <algorithm>
+#include <limits>
+#include <random>
+
+#include "../include/data_structures/message.hpp"
+
+using namespace cadmium;
+using namespace std;
+
+//Port definition
+struct receiver_defs{
+    struct out : public out_port<message_t> {};
+    struct in : public in_port<message_t> {};
+};
+   
+template<typename TIME>
+class Receiver{
+    using defs = receiver_defs; // putting definitions in context
+    public:
+        //Parameters to be overwriten when instantiating the atomic model
+        TIME PREPARATION_TIME;
+        // default constructor
+        Receiver() noexcept{
+            PREPARATION_TIME  = TIME("00:00:10");
+            state.ackNum      = 0;
+            state.sending     = false;
+        }
+        
+        // state definition
+        struct state_type{
+            int ackNum;
```

Comment:
ackNum is a variable. It should all lower case according to Coding Convention 1.

Reply:
Fixed it. Thanks.